components/eamxx/src/physics/share/physics_test_data.cpp

```diff
@@ -26,4 +26,21 @@ PhysicsTestData& PhysicsTestData::assignment_impl(const PhysicsTestData& rhs)
   return *this;
 }
 
+void PhysicsTestData::read(const ekat::FILEPtr& fid)
```

Comment:
At some point we should get rid of the ekat FILEPtr data structure and work directly with fstream from the std library...